### ISSUE TITLE
Maintain a minimum contrast ratio on text

### ISSUE THREAD

**Issue by ewinslow** [Elgg/Elgg#5713]
We have some controls/links which have very light text on white background. This makes them hard to see, breaking accessibility.
- [ ] The login menu (Register, Forgot password, etc.)
- [ ] The top/up/down/bottom controls on admin/plugins

**Comment by jdalsem** [comment]
As themes are mostly custom build i believe we should not try to be better than the rest. We have just a few issue regarding contrast. I would like to focus more on an incorrect navigation/heading structure and missing alt attributes or form element labels than contrast ratios in our default theme.
